pkg/controller/dependences.go

```diff
@@ -314,6 +320,11 @@ func newDependencies(
 		ingv1beta1Lister = kubeInformerFactory.Extensions().V1beta1().Ingresses().Lister()
 	}
 
+	cfg, err := config.LoadDefaultConfig(context.Background())
+	if err != nil {
+		return nil, fmt.Errorf("can't load aws config: %w", err)
+	}
+
```

Comment:
may not return err to be compatible with OP environment.

Reply:
check feature gates before loading aws config?